docs/DebuggingTheCompiler.md

```diff
@@ -582,6 +583,20 @@ it's quite easy to do this manually:
      did wrong. To simplify the comparison, it's sometimes helpful to replace
      all SIL values (e.g. `%27`) with a constant string (e.g. `%x`).
 
+### Bisecting on LLVM optimizer pass counts to identify optimizer bugs
+
+One can use the tips from the previous section to also bisect the LLVM optimizer
+with slightly different options:
+
+1. `-Xllvm --opt-bisect-limit=<INT>`: Maximum number of optimizations to perform. The equivalent of `-sil-opt-pass-count`.
```

Comment:
Should we add some text here on when the verifier is run, how this interacts with Swift-specific passes and so on? By itself, this sounds like this it should be able to bisect across all passes (including Swift-specific ones) but that's not true.